### DISCUSSION TITLE
How do I set the application name on Android?

### DISCUSSION THREAD

**Discussion by polymath74** [AvaloniaUI/Avalonia#16292] [Q&A] [answered]
I have an app started using the avalonia.xplat template. When I run my app on Android, it shows up in the app list as MyAppName.Android but this is not what I want. How can I change this name?

I have tried setting `Name="..."` in App.axaml - changes on desktop but not Android.
I have tried changing `android.label="..."` in AndroidManifest.xml - no effect - this label doesn't actually have ".Android" on the end anyway.

**Comment by timunie** [comment]
Maybe check also csproj file for it. I'm not sure either but worth a look

**Comment by maxkatz6** [comment, marked as answer]
Double check if attribute on MainActivity doesn't override your manifest setting. https://github.com/AvaloniaUI/avalonia-dotnet-templates/blob/master/templates/csharp/xplat/AvaloniaTest.Android/MainActivity.cs#L10

**Reply by polymath74** [reply]
Aha! That was it. Thanks @maxkatz6 :)
